### ISSUE TITLE
Inline support

### ISSUE THREAD

**Issue by AndyObtiva** [luncheon/clocklet#12]
Inline support would be appreciated just like how jQuery-UI Datepicker supports an inline option: https://jqueryui.com/datepicker/#inline

jQuery-UI Timepicker follows a similar API, embedding inline when using 'div' instead of 'input':
https://fgelinas.com/code/timepicker/

**Comment by luncheon** [comment]
It seems to be a duplicate issue with #8.
And it seems to be a common need. So I will provide an API to render the clock without popup.
But for now, please refer to the issue #8. You can find a solution.

**Comment by luncheon** [comment]
Inline rendering API `clocklet.inline(containerElement)` is now available in v0.3.0.

- Standalone example:
https://codepen.io/luncheon/pen/XWKGRro
```js
clocklet
  .inline(document.getElementById('timepicker'))
  .value("1:23 pm")
```

- Example of binding with an `input` element:
https://codepen.io/luncheon/pen/gOMEWbg
```js
clocklet.inline(
  document.getElementById('timepicker'),
  {
    input: document.getElementById('input-time'),
    format: '_H:mm'
  }
)
```

**Comment by AndyObtiva** [comment]
Thank you for the swift response. I just tried to add clocklet to my project [Nebula CDateTime Glimmer Custom Widget](https://github.com/AndyObtiva/glimmer-cw-cdatetime-nebula) for web support and it worked in both inline and dropdown modes. Cheers.
